### ISSUE TITLE
Cannot run `stylelint` after `postcss-nested` - Error:  Cannot read property \'0\' of undefined

### ISSUE THREAD

**Issue by MadLittleMods** [stylelint/stylelint#372]
`stylelint` is throwing an error if used after [`postcss-nested`](https://github.com/postcss/postcss-nested).

```
TypeError: Cannot read property \'0\' of undefined
    at check (/Users/Eastwood/Documents/workspace/spscompanies-master-git/node_modules/stylelint/dist/rules/block-closing-brace-newline-before/index.js:54:31)
    // ...
```

### Setup:

```js
var postcss = require('gulp-postcss');
var nested = require('postcss-nested');
var stylelint = require('stylelint');
var browserReporter = require('postcss-browser-reporter');

postcss([
    nested(),
    stylelint({
        rules: {
            'block-closing-brace-newline-before': [2, 'always']
        }
    }),
    browserReporter()
])
```

```css
a {
    &:hover {
        color: inherit;
    }
}
```

**Comment by jeddy3** [comment]
@davidtheclark I can replicate this. Do you any ideas what it might be? Perhaps `postcss-nested` is mashing the `raws` somehow?

**Comment by davidtheclark** [comment]
Damn, stuff like this makes me want to drop ES6 and just write straight ES5. Not exactly sure which line the error refers to. I'll have to troubleshoot more this evening when I'm back from work.

**Comment by davidtheclark** [comment]
@ai Any ideas about this? It seems that postcss-nested is making `node.raws.after` undefined?

**Comment by ai** [comment]
Any raws property can be missed. Always expect it.

For example, node can be built manually without any raw property.

So, you should use `node.raw('after')` instead.

**Comment by davidtheclark** [comment]
Thanks @ai!

**Comment by jeddy3** [comment]
Good stuff. Does that makes this just a find-and-replace jobby across files?

**Comment by davidtheclark** [comment]
No, I started work on this already. I will have PR soon I hope.

**Comment by davidtheclark** [comment]
@ai: If I do have `raws.after`, should `raws.after` always be the same as `raw("after")`?

If so, I think I found a bug. I have a declaration whose `raws.after` is `undefined` but `raw("after")` is `\n`. Does that make sense to you, or should I file an issue?

**Comment by ai** [comment]
`raw()` use `raws` or autodetect prroperty by other nodes. So if node has no `raws.after`, `raw('after')` will be different.

**Comment by davidtheclark** [comment]
@MadLittleMods : Hopefully this is all taken care of by https://github.com/stylelint/stylelint/pull/376. If you feel like testing it out, you could try the current master branch. Thanks!

**Comment by davidtheclark** [comment]
Released in 1.1.0
